packages/server/eslint.config.mjs

```diff
@@ -38,6 +38,12 @@ const configs = [
       }
     },
     rules: {
+      'no-restricted-imports': [
```

Comment:
Why is this only assigned to '*.ts' files?

Reply:
I assume at some point we will move all the files to .ts right?
Do we want to enforce the rule also in js files?